lib/v2/acg17/post.js

```diff
@@ -0,0 +1,27 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+const host = 'http://acg17.com/';
+
+module.exports = async (ctx) => {
+    const response = await got(`${host}/wp-json/wp/v2/posts?per_page=30`);
```

Comment:
```suggestion
const host = 'http://acg17.com';

module.exports = async (ctx) => {
    const response = await got(`${host}/wp-json/wp/v2/posts?per_page=30`);
```
The URL will have double slash